ktor-client/ktor-client-tests/jvm/src/io/ktor/client/tests/utils/TestServer.kt

```diff
@@ -22,7 +21,7 @@ internal fun startServer(): Closeable {
 
     val proxyServer = TestTcpServer(HTTP_PROXY_PORT, ::proxyHandler)
 
-    val server = embeddedServer(Netty, DEFAULT_PORT) {
+    val server = embeddedServer(CIO, DEFAULT_PORT) {
```

Comment:
Why do you change to CIO everywhere in tests? How is it related to the fix of empty multipart body?

Reply:
This test strikes only with `CIO`. `Netty` ignores wrong prepared request